src/SIL.XForge.Scripture/Services/ParatextNotesMapper.cs

```diff
@@ -265,6 +268,11 @@ string answerExportMethod
                 contentElem.Add(new XElement("p", responseText));
             }
             commentElem.Add(contentElem);
+            if (tagId != NoteTag.notSetId)
```

Comment:
I notice that existing answers to not have this tag added (they still have the red checkbox for me), although new answers made after running this change have the new icon. Is that something we want to do?

Reply:
It looks like this is something we forgot to write in the spec.

My initial thought is I think we should update all existing answers to use the proper note tag, since that seems more helpful. Otherwise existing projects wouldn't get their existing answers brought along to the new note tag, and now their answers are spread across two tags with two icons.

However, that could cause a problem if users want to be able to change the note tag on an answer. If we overwrite it on every sync, and a user had deliberately changed what tag was assigned to a note, that could be annoying. (For example, perhaps a note tag called "Answers that we should act upon" would be created, and some answers would be changed to use that tag, so they could be filtered for in Paratext).

We could solve both of these by checking if the note ever had the expected tag, and if never did, update it to the correct tag. But if it used to have the correct tag, but doesn't anymore, then assume a user changed it in Paratext and leave it alone. But that's more work, and depending on how complicated it is to check the history of the note, it may or may not be worth it.

Reply:
Actually, as I was investigating this I found a simple way to retroactively set the note tag. We will always set the answer tag to the current community checking tag, but if a user in Paratext decides to change the tag, a new comment is created to represent the change. Because of how that works, if we retroactively always set the answer tag to be the current checking tag, subsequent tag changes are unaffected.

There may be an argument that we do not want to always change existing answers to have the current tag, but I would argue that we DO want this and that when a user decides that the purple convo icon isn't what they want, they should expect that when they synchronize with SF, all existing answers will get updated to the new tag.